docs/src/components/Demo.tsx

```diff
@@ -16,14 +37,16 @@ export const Demo = ({ children, propControls, themeControls, code }) => {
   return (
     <Card className="docs-component-demo">
       <Flex direction="row" alignItems="stretch">
-        <Flex direction="column" width="50%">
-          {children}
+        <Flex direction="column" flex="1">
+          <View>{children}</View>
```

Comment:
not a blocker: why do we need to enclose children within a View?

Reply:
Both this, and Scott's comment below are because of wrapping things in a Flex kind of breaks the inline/block model. If we set `alignItems` to `flex-start` then any element that is normally a block, like an Alert because it is a div, gets shrunk to its contents width. If it is set to 'normal' or 'stretch' then it will stretch inline-block elements like the badge. There might be some magical combination of properties to get both scenarios to work, but wrapping the contents in a div unsets the flex-iness of its children